bodhi/tests/server/consumers/test_consumers.py

```diff
@@ -0,0 +1,53 @@
+import mock
+
+from bodhi.tests.server import base
+from fedora_messaging.api import Message
```

Comment:
For PEP-8, this should be in the third party block above the Bodhi imports and below the suggested mock import above.